wondrous-app/components/BountySection/modal.tsx

```diff
@@ -0,0 +1,173 @@
+import { useLazyQuery } from '@apollo/client';
+import { EXPLORE_MODAL_TABS_MAP, LINK } from 'utils/constants';
+import React, { useEffect, useState } from 'react';
+import Modal from '@mui/material/Modal';
+import { CircularProgress } from '@mui/material';
+import { GET_GR15_GRANTEES, GET_GR15_SPONSORS } from 'graphql/queries/org';
+import { TaskModalBaseCard } from 'components/Common/Task/styles';
+import CloseModalIcon from 'components/Icons/closeModal';
+import {
+  ActivityIndicatorContainer,
+  CloseIconContainer,
+  Container,
+  ExploreStyledTabs,
+  OverflowBox,
+  StyledTab,
+  TabText,
+  Title,
+  TitleSection,
+} from 'components/profile/modals/styles';
+
+import { SafeImage } from 'components/Common/Image';
+import { TokenEmptyLogo } from 'components/organization/wrapper/styles';
+import { DAOEmptyIcon } from 'components/Icons/dao';
+import { OrgItemContainer, OrgItemDescription, OrgItemDescriptionDiv, OrgItemDescriptionTitle } from './styles';
+
+const OrgItem = (props) => {
+  const { org } = props;
+  return (
+    <OrgItemContainer
+      onClick={() => {
+        window.open(`${LINK}/organization/${org?.username}/boards?entity=task`);
+      }}
+    >
+      <SafeImage
+        src={org?.profilePicture}
+        placeholderComp={
+          <TokenEmptyLogo>
+            <DAOEmptyIcon />
+          </TokenEmptyLogo>
+        }
+        width="42px"
+        height="42px"
+        useNextImage
+        style={{
+          borderRadius: '6px',
+        }}
+      />
+      <OrgItemDescriptionDiv>
+        <OrgItemDescriptionTitle>{org?.name}</OrgItemDescriptionTitle>
+        <OrgItemDescription>{org?.description}</OrgItemDescription>
+      </OrgItemDescriptionDiv>
+    </OrgItemContainer>
+  );
+};
+
+export default function ExploreOrgGr15Modal(props) {
+  const { showSponsors, showGrantees, setShowSponsors, setShowGrantees, open, handleClose, name } = props;
+  const [listLoading, setListLoading] = useState(true);
+  const [sponsorList, setSponsorList] = useState([]);
+  const [granteeList, setGranteeList] = useState([]);
+  const [activeTab, setActiveTab] = useState(null);
+  const [getGr15Sponsors] = useLazyQuery(GET_GR15_SPONSORS, {
+    onCompleted: (data) => {
+      const sponsorData = data.getGr15Sponsors;
+      setSponsorList(sponsorData || []);
+      setListLoading(false);
+    },
+    fetchPolicy: 'cache-and-network',
+  });
+
+  const [getGr15Grantees] = useLazyQuery(GET_GR15_GRANTEES, {
```

Comment:
I think we can get rid of state here ( granteeList & sponsorList ) and use data?.getGr15Grantees since we're never resetting this data

Then on the render we can just data?.getGr15Grantees / data?.getGr15Sponsors